.github/workflows/test.yml

```diff
@@ -19,19 +19,15 @@ on:
 
 jobs:
   test:
-
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: ['windows-latest', 'macOS-latest', 'ubuntu-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
-        exclude:  # Python < 3.8 is not supported on Apple Silicon ARM64
-          - os: macOS-latest
-            python-version: '3.7'
-        include:  # So run on older version on Intel CPU
-          - os: macOS-13
-            python-version: '3.7'
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
+        exclude:
+          - os: ubuntu-latest
+            python-version: '3.12'
```

Comment:
Any reason for excluding this?

Reply:
Because the tests keep randomly hanging on python3.12 and ubuntu (without providing an error)

Reply:
![image](https://github.com/user-attachments/assets/19f5b55e-f674-4dc5-970b-58df4b60213d)